content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -114,7 +231,28 @@ Create the authorization policy:
       rules:
       - from:
         - source:
-           ipBlocks: ["1.2.3.4", "5.6.7.0/24", "$CLIENT_IP"]
+            ipBlocks: ["1.2.3.4", "5.6.7.0/24", "$CLIENT_IP"]
+    EOF
+    {{< /text >}}
+
+1. With `remoteIpBlocks`:
```

Comment:
also probably use tab here, I think we want the user to choose one of {ipBlocks, remoteIpBlocks} but not both.